packages/trader/src/Stores/Modules/Trading/trade-store.js

```diff
@@ -780,6 +790,7 @@ export default class TradeStore extends BaseStore {
                             this.debouncedProposal();
                             this.clearLimitOrderBarriers();
                             this.pushPurchaseDataToGtm(contract_data);
+                            this.getActionToastbox(response.buy);
```

Comment:
This doesn't get anything. It's more of a set

Reply:
yes will rename it